components/raftstore-v2/src/batch/store.rs

```diff
@@ -373,9 +373,9 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
         workers
             .store_writers
             .spawn(store_id, raft_engine.clone(), None, router, &trans, &cfg)?;
-        let log_fetch_scheduler = workers.log_fetch_worker.start(
-            "raftlog-fetch-worker",
-            RaftlogFetchRunner::new(router.clone(), raft_engine.clone()),
+        let read_scheduler = workers.async_read_worker.start(
+            "async-read-worker",
```

Comment:
We should check if any metrics/alerts depends on the name.

Reply:
Only change it for v2.

Reply:
V2 also reuse the grafana dashboard.

Reply:
Only one metric about raftlog's pending fetch task. I added it in the grafana tmpl.